projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -817,7 +817,10 @@ class_exists( 'Jetpack_AMP_Support' )
 	 * @param string $content HTML content of the post.
 	 * @return string
 	 */
-	public function add_data_img_tags_and_enqueue_assets( $content ) {
+	public function add_data_img_tags_and_enqueue_assets( $content = '' ) {
```

Comment:
Does it really make sense to set a default here? i.e. are we getting fatals because some caller doesn't pass the arg at all?

Note that a caller explicitly passing null won't be affected by a default. The check below will be handling that.

Reply:
You're right; it would've been a different error if they didn't pass any args. I was covering both scenarios, but don't see any evidence of the no-arg scenario happening. How about c670f5dd3e?

Reply:
:+1: